tests/integration/ps.bats

```diff
@@ -21,8 +21,39 @@ function teardown() {
 
   testcontainer test_busybox running
 
-  runc ps test_busybox -eaf
+  runc ps test_busybox
```

Comment:
Can we keep `-eaf` here? I'm planning on updating the busybox image in the console PR, and the default `ps` format changed between 2014 and 2016.